ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -87,40 +77,96 @@ public GanttCSVExport(IGanttProject project, CSVOptions csvOptions) {
     myRoleManager = Preconditions.checkNotNull(roleManager);
     this.csvOptions = Preconditions.checkNotNull(csvOptions);
   }
+
+
+
+  private CSVFormat getFormatForWriter() {
+      CSVFormat format = CSVFormat.DEFAULT.withEscape('\\');
+      if (csvOptions.sSeparatedChar.length() == 1) {
+          format = format.withDelimiter(csvOptions.sSeparatedChar.charAt(0));
+      }
+      if (csvOptions.sSeparatedTextChar.length() == 1) {
+          format = format.withQuote(csvOptions.sSeparatedTextChar.charAt(0));
+      }
+
+      return format;
+  }
+
+  private CommonWriter getWriter (OutputStream stream, String formatName) throws IOException {
+    if (formatName!=null && formatName.length()>0) {
```

Comment:
add spaces around `!=` and `>`
Also, is this check really needed? It must be error if format is null or empty, and code should not just ignore this error. I suggest replacing it with 
```
assert (formatName != null && !"".equals(formatName)) : "Wrong format";
```

or even better, pass enum value here, so that there were just no options